docs/DeveloperGuide.md

```diff
@@ -274,6 +274,10 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | -------- | ----------------------------| ------------------------------------------------------------- | ------------------------------------------------------------------------------|
 | `* * *`  | forgetful/busy nurse        | get reminders of medical needs of those under my care         | be reminded of my duties that I may otherwise forget                          |
 | `* * *`  | nurse                       | view all the elderly I am in charge of/added into NurseyBook  | have a full list of elderly for easy reference                                |
+| `* * *`  | nurse                       | view all tasks                                                | I can have an overview of all my tasks                                        |
+| `* * *`  | nurse                       | add a one-off task                                            | I can keep track of what I have to do                                         |
+| `* * *`  | nurse                       | delete a task                                                 | I can have a cleaner workspace                                                |
+| `* * *`  | nurse                       | mark a task as completed                                      | I will not receive reminders that are no longer relevant                      |
```

Comment:
For the code added/changed in lines 277 to 280, I think it would be better to remove the 'I can' before the 'So that I can... ' section, as it has been included in the header.